client.go

```diff
@@ -170,3 +171,92 @@ func Exec(ctx context.Context, cmd string, args ...interface{}) error {
 func Query(ctx context.Context, cmd string, args ...interface{}) Args {
 	return DefaultClient.Query(ctx, cmd, args...)
 }
+
+type MultiError []error
```

Comment:
The type implement the `errror` interface, you can change the signature of your functions to return `error` instead of `MultiError` and is should just work I think.